app/_hub/kong-inc/prometheus/3.0.0.md

```diff
@@ -0,0 +1,281 @@
+kong_upstream_latency_ms_bucket{service="google",route="google.route-1",le="2000"} 1
+kong_upstream_latency_ms_bucket{service="google",route="google.route-1",le="5000"} 1
+kong_upstream_latency_ms_bucket{service="google",route="google.route-1",le="10000"} 1
+kong_upstream_latency_ms_bucket{service="google",route="google.route-1",le="30000"} 1
+kong_upstream_latency_ms_bucket{service="google",route="google.route-1",le="60000"} 1
+kong_upstream_latency_ms_bucket{service="google",route="google.route-1",le="+Inf"} 1
+kong_upstream_latency_ms_count{service="google",route="google.route-1"} 1
+kong_upstream_latency_ms_sum{service="google",route="google.route-1"} 2
+
+```
+
+{:.note}
+> **Note:** Upstream targets' health information is exported once per subsystem. If both
+stream and HTTP listeners are enabled, targets' health will appear twice. Health metrics
+have a `subsystem` label to indicate which subsystem the metric refers to.
+
+## Accessing the metrics
+
+In most configurations, the Kong Admin API will be behind a firewall or would
+need to be set up to require authentication. Here are a couple of options to
+allow access to the `/metrics` endpoint to Prometheus:
+
+
+1. If the [Status API](/gateway/latest/reference/configuration/#status_listen)
+   is enabled, then its `/metrics` endpoint can be used.
+   This is the preferred method.
+
+1. The `/metrics` endpoint is also available on the Admin API, which can be used
+   if the Status API is not enabled. Note that this endpoint is unavailable
+   when [RBAC](/gateway/latest/admin-api/rbac/reference/) is enabled on the
+   Admin API (Prometheus does not support Key-Auth to pass the token).
+
+---
+
+## Changelog
+
+* Metric names have been adjusted to add units to better standardize where possible
+  * `http_status` -> `http_requests_total`
+  * `latency` -> `kong_request_latency_ms`/`kong_upstream_latency_ms`/`kong_kong_latency_ms`
+  * `kong_bandwidh` -> `kong_bandwidth_bytes`
+  * `nginx_http_current_connections`/`nginx_stream_current_connections` -> `nginx_hconnections_total`
+  * `http_consumer_status` removed.
+* New Metric: `session_duration_ms` for monitoring stream connections.
+* New Metric: `node_info`. Single gauge set to 1 that outputs the node's id and kong version.
+* Latency has been split into 4 different metrics: `kong_latency_ms`, `upstream_latency_ms` and `request_latency_ms` (http) /`session_duration_ms` (stream). Buckets details below.
+  * Kong Latency and Upstream Latency can operate at orders of magnitudes different. Separate out these buckets to reduce memory overhead.
+* `request_count` and `consumer_status` have been merged into just `http_requests_total`. If the `per_consumer` config is set false, the `consumer` label will be empty.  If the `per_consumer` config is true, it will be filled.
+* `http_requests_total` has a new label `source`. set to either `exit`, `error` or `service`. https://docs.konghq.com/gateway/latest/pdk/kong.response/#kongresponseget_source
```

Comment:
```suggestion
* `http_requests_total` has a new label [`source`](/gateway/latest/pdk/kong.response/#kongresponseget_source). It can be set to `exit`, `error`, or `service`.
```